### ISSUE TITLE
cannot adjust volume in iOS 15

### ISSUE THREAD

**Issue by faslurrajah** [cheebeez/volume_regulator#3]
I can move the slider in iOS simulator

**Comment by cheebeez** [comment]
Yes, the simulator does not process all events, unlike a real device.

**Comment by faslurrajah** [comment]
please update it to README.  I wasted hours to figure it out.

**Comment by faslurrajah** [comment]
Yes, it's not. But it may take time to understand about simulator issue. Having it in README is better
